TKeazirian.HTTPServer/Router/RoutesConfig.cs

```diff
@@ -1,30 +1,33 @@
+using TKeazirian.HTTPServer.Response;
+
 namespace TKeazirian.HTTPServer.Router;
 
 using Handler;
+using static HttpMethods;
 
-public class RoutesConfig
+public static class RoutesConfig
 {
-    public readonly Dictionary<string, Handler> Routes;
-
-    public RoutesConfig()
+    public static Routes ConstructRoutes()
     {
-        Routes =
-            new Dictionary<string, Handler>
-            {
-                { "/echo_body", new EchoBodyHandler() },
-                { "/simple_get", new SimpleGetHandler() },
-                {
-                    "/simple_get_with_body", new SimpleGetHandler()
-                },
-                { "/redirect", new RedirectHandler() },
-                { "/head_request", new SimpleHeadHandler() },
-                { "/method_options", new SimpleOptionsHandler() },
-                { "/method_options2", new SimpleOptionsHandler2() },
-            };
-    }
+        Routes routes = new();
 
-    public RoutesConfig(Dictionary<string, Handler> routes)
-    {
-        Routes = routes;
+        routes.AddRoute(
+            new("/simple_get", new() { Get }, new SimpleGetHandler())
+        );
+        routes.AddRoute(
+            new("/simple_get_with_body", new() { Get }, new SimpleGetHandler())
+        );
+        routes.AddRoute(
+            new("/echo_body", new() { Post }, new EchoBodyHandler()));
+        routes.AddRoute(
+            new("/redirect", new() { Get }, new RedirectHandler()));
+        routes.AddRoute(
+            new("/head_request", new() { Get }, new SimpleHeadHandler()));
+        routes.AddRoute(
+            new("/method_options", new() { Get }, new SimpleOptionsHandler()));
+        routes.AddRoute(
+            new("/method_options2", new() { Get, Put, Post },
+                new SimpleOptionsHandler()));
+        return routes;
```

Comment:
This looks much more like what I'd expect to see in a routes configuration! It's kind of similar to Express, where you'd defined it something like `router.get("/simple_get", simpleGetHandler)`